tools/prologs-epilogs/gpu-health

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+#
```

Comment:
I suggest you fail gracefully if `nvidia-smi` or `dcgmi` do not exist. Examples:

```shell
if ! type -P nvidia-smi 1>/dev/null; then
        # this script requires nvidia-smi to function
        exit 0
fi
```

@samskillman WDYT?